src/app/tests/TestMessageDef.cpp

```diff
@@ -28,6 +28,8 @@
 #include <app/MessageDef/InvokeCommand.h>
 #include <app/MessageDef/ReadRequest.h>
 #include <app/MessageDef/ReportData.h>
+#include <app/MessageDef/SubscribeRequest.h>
```

Comment:
should we consider splitting the tests into separate classes? We have the messagedef already split however the test is still a monolith

Reply:
https://github.com/project-chip/connectedhomeip/issues/8220
Agree, would handle it in separate ticket, thanks